medicines/web/components/pdf/index.tsx

```diff
@@ -33,19 +34,27 @@ const Pdf: React.FC = () => (
     <h2>{title}</h2>
     <ul>
       <li>
-        <a href={helpUrl} target="_new">
+        <ReactGA.OutboundLink eventLabel={helpLabel} to={helpUrl} target="_new">
```

Comment:
Is `ReactGA.OutboundLink` required on all external links?

There are some in the footer, about page, and the Yellow Card.

Reply:
It would be ideal but these ones specifically were asked to be tracked.